src/constants/routesPaths.js

```diff
@@ -0,0 +1,11 @@
+const routes = {
+    index: '/',
+    discover: '/discover',
+    trending: '/trending',
+    wishlist: '/wishlist',
+    topRated: '/toprated',
+    movieDetail: '/movie/:id'    
+  };
```

Comment:
I guess you can reuse this constants for the links that I mention before. you'll need to add another one for generating routes for movieDetail.